internal/apischema/openai/openai_test.go

```diff
@@ -164,7 +164,7 @@ func TestOpenAIChatCompletionMessageUnmarshal(t *testing.T) {
 			in: []byte(`{"model": "gpu-o4",
                         "messages": [
                          {"role": "assistant", "content": "you are a helpful assistant"},
-                         {"role": "assistant", "content": "{'text': 'you are a helpful assistant'}"}
```

Comment:
I think the previous code wasn’t valid JSON in the first place (maybe a hallucination?).

Reply:
I think this gets decoded into the singular assistant content type which you have fixed with the array type.